kyuubi-server/src/test/scala/org/apache/kyuubi/server/api/v1/BatchesResourceSuite.scala

```diff
@@ -852,4 +852,92 @@ abstract class BatchesResourceSuiteBase extends KyuubiFunSuite
     val getBatchListResponse = response.readEntity(classOf[GetBatchesResponse])
     assert(getBatchListResponse.getTotal == 1)
   }
+
+  test("open batch session with proxyUser") {
+    // we use superUser to impersonate commonUser
+    val superUser = "superUser"
+    val commonUser = "commonUser"
+
+    val openBatchExecutor = (kyuubiProxyUser: String, hs2ProxyUser: String) => {
```

Comment:
some style suggestion about writing scala code
```suggestion
    def runOpenBatch = (kyuubiProxyUser: Option[String], hs2ProxyUser: Option[String]) => {
```